scene/2d/physics_body_2d.cpp

```diff
@@ -1095,11 +1088,11 @@ void KinematicBody2D::set_sync_to_physics(bool p_enable) {
 	}
 }
 
-bool KinematicBody2D::is_sync_to_physics_enabled() const {
+bool CharacterBody2D::is_sync_to_physics_enabled() const {
```

Comment:
Sync to physics should most likely be moved to StaticBody2D/3D, since its aim is mostly to implement stuff like moving platforms. For characters, this is most likely a given, as they always have linear/angular velocity.

Reply:
Nevermind the last commit fixes this.